src/opl.c

```diff
@@ -448,6 +450,12 @@ int oplScanApps(int (*callback)(const char *path, config_set_t *appConfig, void
                 LOG("APPS failed to open dir %s\n", appsPath);
         }
     }
+    mc++;
+    if (mc < 2)
+    {
+        snprintf(appsPath, sizeof(appsPath), "mc%d:", mc);
+        goto readDir;
```

Comment:
Never use 'goto' unless there's no other option (hint; there always is). Forget goto exists. Please find a different solution.